ios/Classes/BetterPlayer.m

```diff
@@ -762,6 +766,9 @@ - (void)pictureInPictureControllerDidStartPictureInPicture:(AVPictureInPictureCo
 }
 
 - (void)pictureInPictureControllerWillStopPictureInPicture:(AVPictureInPictureController *)pictureInPictureController  API_AVAILABLE(ios(9.0)){
+    if (_isPreparingDatasource) {
```

Comment:
https://github.com/jhomlala/betterplayer/blob/master/ios/Classes/BetterPlayer.m#L442
Sorry. We cannot check like that?

Reply:
I'm sorry, I used this method to check it, and it's not working. The status it returns is AVPlayerStatusReadyToPlay.

Reply:
Thank you for checking it. 👍 
I understand.